app/views/govuk_publishing_components/components/_checkboxes.html.erb

```diff
@@ -15,11 +15,11 @@
         <% end %>
 
         <% if cb_helper.hint_text %>
-          <%= tag.span cb_helper.hint_text, id: "#{id}-hint", class: "govuk-hint" %>
+          <%= tag.div cb_helper.hint_text, id: "#{id}-hint", class: "govuk-hint" %>
         <% end %>
 
         <% if cb_helper.error %>
-          <%= tag.span error, id: "#{id}-error", class: "govuk-error-message" %>
+          <%= tag.p error, id: "#{id}-error", class: "govuk-error-message" %>
```

Comment:
@andysellick 

----
The link in the commit message is slightly wrong, should be `https://github.com/alphagov/govuk-frontend/blob/main/CHANGELOG.md#update-the-html-for-error-messages`